.github/actions/find-changed-files/src/main.ts

```diff
@@ -19,46 +15,12 @@ export async function main() {
   const inputs = getInputs();
   core.info(`Received input: ${JSON.stringify(inputs, null, 2)}`);
 
-  const res = await findChangedFiles(inputs);
+  const res = await findChangedFiles(inputs.githubToken);
 
   core.info(`Result: ${JSON.stringify(res, null, 2)}`);
   core.setOutput('all_modified_files', res.allModifiedFiles.join(','));
 }
 
-export async function findChangedFiles(inputs: Inputs): Promise<Output> {
```

Comment:
can this be used to replace tj-actions?

Reply:
Partially, it currently only reimplements some of the functionality from `tj-actions-changed-files`, but we can pretty easily extend it to cover more of that functionality as needed

Reply:
Specifically, it only gives the equivalent of this output: https://github.com/tj-actions/changed-files/blob/27ae6b33eaed7bf87272fdeb9f1c54f9facc9d99/action.yml#L301